paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -915,17 +922,264 @@ class PaymentSheet internal constructor(
             primaryButton = PrimaryButton(),
         )
 
+        constructor(
+            colorsLight: Colors = Colors.defaultLight,
+            colorsDark: Colors = Colors.defaultDark,
+            shapes: Shapes = Shapes.default,
+            typography: Typography = Typography.default,
+            primaryButton: PrimaryButton = PrimaryButton(),
+        ) : this(
+            colorsLight = colorsLight,
+            colorsDark = colorsDark,
+            shapes = shapes,
+            typography = typography,
+            primaryButton = primaryButton,
+            embeddedAppearance = Embedded.default
+        )
+
         fun getColors(isDark: Boolean): Colors {
             return if (isDark) colorsDark else colorsLight
         }
 
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        @Parcelize
+        class Embedded(
+            internal val style: RowStyle
+        ) : Parcelable {
+
+            internal companion object {
+                val default = Embedded(
+                    style = RowStyle.FlatWithRadio.defaultLight
+                )
+            }
+
+            @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+            @Parcelize
+            sealed class RowStyle : Parcelable {
+
+                @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+                @Parcelize
+                class FlatWithRadio(
+                    /**
+                     * The thickness of the separator line between rows.
+                     */
+                    val separatorThicknessDp: Float,
```

Comment:
Should these have default values?

Reply:
Happy to discuss as a follow up too.

Reply:
Yeah I can update to match iOS' defaults in the next PR